modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/result/display/context/builder/SearchResultSummaryDisplayContextBuilder.java

```diff
@@ -891,11 +890,16 @@ private void _buildViewURL(
 	}
 
 	private String _formatDate(Date date) {
-		Format format = _fastDateFormatFactory.getDateTime(
-			FastDateFormatConstants.MEDIUM, FastDateFormatConstants.SHORT,
-			_locale, _themeDisplay.getTimeZone());
+		DateFormat dateTimeFormat = DateFormat.getDateTimeInstance(
+			DateFormat.LONG, DateFormat.SHORT, _locale);
```

Comment:
@MrJohn1911 Let's focus on fixing the test and not change the production code on this one.
Note that you are using DateFormat.LONG instead of MEDIUM now.